types/indy-sdk/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false
```

Comment:
Why is this needed?

Reply:
The ci was giving me errors and this was what I could find to fix it. However I've made some changes afterwards, so maybe it will also work without it. I've reverted the change to see if the pipeline will pass

Reply:
Seems to work. Thanks!